example/advection/main.cpp

```diff
@@ -0,0 +1,53 @@
+//========================================================================================
+// (C) (or copyright) 2020. Triad National Security, LLC. All rights reserved.
+//
+// This program was produced under U.S. Government contract 89233218CNA000001 for Los
+// Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+// for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+// in the program are reserved by Triad National Security, LLC, and the U.S. Department
+// of Energy/National Nuclear Security Administration. The Government is granted for
+// itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works, distribute copies to
+// the public, perform publicly and display publicly, and to permit others to do so.
+//========================================================================================
+
+// C headers
+
+// C++ headers
+
+// Parthenon headers
+#include "parthenon_manager.hpp"
+
+// Application headers
+#include "advection.hpp"
+
+int main(int argc, char *argv[]) {
+  using parthenon::ParthenonManager;
+  using parthenon::ParthenonStatus;
+  ParthenonManager pman;
+
+  auto manager_status = pman.ParthenonInit(argc, argv);
+  if (manager_status == ParthenonStatus::complete) {
+    pman.ParthenonFinalize();
+    return 0;
+  }
+  if (manager_status == ParthenonStatus::error) {
+    pman.ParthenonFinalize();
+    return 1;
+  }
+
+  AdvectionDriver driver(pman.pinput.get(), pman.pmesh.get(), pman.pouts.get());
+
+  // start a timer
+  pman.PreDriver();
+
+  auto driver_status = driver.Execute();
+
+  // Make final outputs, print diagnostics
+  pman.PostDriver(driver_status);
+
+  // call MPI_Finalize if necessary
+  pman.ParthenonFinalize();
```

Comment:
So no `Kokkos::Views` can be created before `ParthenonInit` and they must all be gone by `ParthenonFinalize`. I realize that the `init` and `finalize` methods ensure this happens, but I'd still feel more comfortable if they wrapped scope. Especially for an example.

Reply:
I'm not sure I understand what you have in mind here.  Do you want { } somewhere?  We could put them between PathenonInit and ParthenonFinalize, but that's a lie.  ParthenonInit, in particular, will certainly have new ```Kokkos::Views``` created.

Reply:
Yeah i wanted them between Parthenon init and Parthenon finalize.

Maybe we don’t need that. Instead perhaps a comment explaining where kokkos init and finalize are and what that implies about views would suffice.